src/t09/calculations/calculationT09D.js

```diff
@@ -34,8 +33,8 @@ export function calcXt(k, b, q, Qw, xw, rhof, rhos) {
         const term2 = Qw*Math.log(Math.pow((xt-xw),2)/Math.pow((xt+xw),2))/(4*Math.PI*k);
         rhs = term1 + term2;
         xt = xt + (lhs - rhs);
-        iter = iter + 1;
-    } while (Math.abs(lhs-rhs)>0.01);
+        console.log(xt)
```

Comment:
Can we remove this console.log here?
Possibly there is no need for this while loop?!